packages/frontend/src/main/components/activity/ListItemActivity.vue

```diff
@@ -540,6 +539,10 @@ export default {
     }
   },
   methods: {
+    /**
+     * TODO: no bueno, needs refactoring. As eslint warns us - using v-html can lead to XSS attacks.
+     * Additionally I don't see why this can't be done with Vue templates
```

Comment:
I'm guessing that the description of streams & branches can be freely edited by the user, right? In that case this is a serious security issue